packages/app-runtime/src/natives/INativeDeviceInfoAccess.ts

```diff
@@ -0,0 +1,23 @@
+import { Result } from "@js-soft/ts-utils";
+
+export enum PushServices {
+    "apns" = "apns", // eslint-disable-line @typescript-eslint/naming-convention
+    "fcm" = "fcm", // eslint-disable-line @typescript-eslint/naming-convention
+    "none" = "none" // eslint-disable-line @typescript-eslint/naming-convention
```

Comment:
Can we add `"dummy"` here? (I remember we did some black magic a while ago to set that... https://github.com/nmshd/runtime/blob/ac65997fdaab9df917f46012e75ee6a57744a583/packages/app-runtime/test/mocks/NativeDeviceInfoAccessMock.ts#L14)

Reply:
Wow, such a pretty formatting from git. 🤩

Reply:
hmmmm, IMO we should do that later, but good catch!

Reply:
no, NOW!